src/main/java/com/collegecompendium/backend/models/College.java

```diff
@@ -37,4 +31,8 @@ public class College extends User {
 	@Min(100)
 	@NotNull
 	private Integer cost;
+	
+	public List<Degree> getDegrees() {
```

Comment:
What is the reasoning behind this? Does Lombok not create the getter correctly for the degrees field>

Reply:
Lombok's getters from @Data did not resolve this issue, had to input it manually.